addons/website_event_booth/static/src/js/booth_register.js

```diff
@@ -54,18 +61,22 @@ publicWidget.registry.boothRegistration = publicWidget.Widget.extend({
     },
 
     _fillBooths() {
-        var $boothElem = this.$('.o_wbooth_booths');
-        $boothElem.empty();
-        $.each(this.boothCache[this.activeType], function (key, booth) {
+        let boothsElem = this.el.querySelector('.o_wbooth_booths');
+        boothsElem.innerHTML = '';
+        let self = this;
+        for (const booth of this.boothCache[this.activeType]) {
+            let selectedBooth = self.boothsFirstRendering && self.selectedBoothIds.includes(booth.id.toString());
             let $checkbox = dom.renderCheckbox({
```

Comment:
Here, renderCheckbox returns a jquery object, so I kept a few while we keep this solution.

Notably .append, maybe we could use insertAdjacentHTML and .outerHTML but while we have it...

Also I am not sure what the recommended solution is when it comes to replacing an element with the result of a Qweb.render